lib/Gitlab/Api/Issues.php

```diff
@@ -105,4 +105,15 @@ public function updateComment($project_id, $issue_id, $note_id, $body)
             'body' => $body
         ));
     }
+
+    /**
+     * @param int $project_id
+     * @param int $issue_id
+     * @param int $note_id
+     * @return mixed
+     */
+    public function deleteComment($project_id, $issue_id, $note_id)
```

Comment:
@chteuchteu Pls rename to `removeComment`, to have consistency compared with other methods within the project. See #149 discussion.